main.go

```diff
@@ -47,17 +48,18 @@ type BootstrapConfig struct {
 
 type RegistryCmd struct {
 	BootstrapConfig
-	ContainerdRegistryConfigPath string        `arg:"--containerd-registry-config-path" default:"/etc/containerd/certs.d" help:"Directory where mirror configuration is written."`
-	MetricsAddr                  string        `arg:"--metrics-addr,required" help:"address to serve metrics."`
-	LocalAddr                    string        `arg:"--local-addr,required" help:"Address that the local Spegel instance will be reached at."`
-	ContainerdSock               string        `arg:"--containerd-sock" default:"/run/containerd/containerd.sock" help:"Endpoint of containerd service."`
-	ContainerdNamespace          string        `arg:"--containerd-namespace" default:"k8s.io" help:"Containerd namespace to fetch images from."`
-	RouterAddr                   string        `arg:"--router-addr,required" help:"address to serve router."`
-	RegistryAddr                 string        `arg:"--registry-addr,required" help:"address to server image registry."`
-	Registries                   []url.URL     `arg:"--registries,required" help:"registries that are configured to be mirrored."`
-	MirrorResolveTimeout         time.Duration `arg:"--mirror-resolve-timeout" default:"5s" help:"Max duration spent finding a mirror."`
-	MirrorResolveRetries         int           `arg:"--mirror-resolve-retries" default:"3" help:"Max amount of mirrors to attempt."`
-	ResolveLatestTag             bool          `arg:"--resolve-latest-tag" default:"true" help:"When true latest tags will be resolved to digests."`
+	BlobSpeed                    *throttle.Byterate `arg:"--blob-speed" help:"Maximum write speed when serving blob layers. Should be number followed by unit Bps, KBps, MBps, GBps, or TBps."`
```

Comment:
Is BlobSpeed meant to be per request or global? My understanding of the code is that it's currently implemented to be per request.

If that's the case, then it should be made clear in the usage of it. If it's meant to be global, maybe we should look into where it's implemented.

Reply:
Blob speed is per request, implementing a global limit would create a lot more complications with syncing between routines. In my head I would assume that speed would be per request and not global, it is also what all other projects do. I will update the help text to make this clear.